<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="anshulmadan01" failures="1" tests="1" name="Backend" time="92.611" errors="0" timestamp="2022-12-08T08:45:11 GMT">
  <testcase classname="com.assignment.Backend.CreateANewUser" name="RegisterNewUser" time="87.101">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	test case is failedTimed out after 4 seconds. Unable to locate the element expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
test case is failedTimed out after 4 seconds. Unable to locate the element expected [true] but found [false]
at com.assignment.Backend.CreateANewUser.RegisterNewUser(CreateANewUser.java:77)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- RegisterNewUser -->
</testsuite> <!-- Backend -->
